content/docs/actors/development/secret_input.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Secret input
+description: Learn about making some input fields secret and encrypted.
```

Comment:
You get $1,000,000 if you make the meta description 120-160 characters (good for SEO)